docs/Installation.md

```diff
@@ -85,10 +85,6 @@ spec:
       labels:
         app: shiori
     spec:
-      securityContext:
```

Comment:
I'm not running mine in k8s anymore, but last time I did this was necessary, did we fix it along the way?

Reply:
Good question on how it got fixed. I tried it in k3s and kind, just to try a more restrictive environment, and it ran without the security context in both.